acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerRunner.java

```diff
@@ -120,6 +120,8 @@ public void shutdown() {
       params.add("--data-path");
       params.add(dataPath.toAbsolutePath().toString());
     }
+    params.add("--slashing-protection-enabled");
```

Comment:
again - take a look at one of my PRs, think this is done with a URL

Reply:
removed it